cpputil/unstructured/include/gridtools/next/mesh.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <cstddef>
+#include <gridtools/common/hymap.hpp>
+
+namespace gridtools {
+    namespace next {
+
+        namespace connectivity {
+            // Doesn't need to be callable on device
+            template <class Connectivity>
+            GT_FUNCTION_HOST auto neighbor_table(Connectivity const &connectivity) {
+                return connectivity_neighbor_table(connectivity);
+            };
+
+            template <class Connectivity>
+            GT_FUNCTION auto max_neighbors(Connectivity const &connectivity) {
+                return connectivity_max_neighbors(connectivity);
+            }
+
+            template <class Connectivity>
+            GT_FUNCTION std::size_t size(Connectivity const &connectivity) {
+                return connectivity_size(connectivity);
+            }
+
+            template <class Connectivity>
+            GT_FUNCTION auto skip_value(Connectivity const &connectivity) {
+                return connectivity_skip_value(connectivity);
+            }
```

Comment:
skip value is known in compile time. So we can do: 
```
template <class C>
decltype(connectivity_skip_value(std::declval<T const&>())) skip_value_helper(C const&);

template <class C, class D = std::decay_t<sid::reference_type<neighbor_table_type<C>>>>
using skip_type = gridtools::integral_constant<D, decltype(::gridtools::next::connectivity::skip_value_helper(declval<C const&>()))::value>;

```

Reply:
I double-checked that in atlas this is runtime information